website_snippet_blog_display_post/models/view_view.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    def is_node_branded(self, node):
```

Comment:
when this was published I secretly hoped for someone to yell that we just utterly misunderstood the snippet engine and that this is a standard that we overlooked. If this actually is an elegant solution, it would be worthwhile to encapsulate it in an own module, probably with some examples.